sdks/python/apache_beam/options/value_provider.py

```diff
@@ -69,6 +69,7 @@ def __eq__(self, other):
 
 class RuntimeValueProvider(ValueProvider):
   runtime_options = None
+  is_time_counter_enabled = False
```

Comment:
@boyuanzz why do you need this to be a special case here? `RuntimeValueProvider` is not the right place to have attributes for different options.

Does it make a difference in performance? Can you cache this somewhere else?

Reply:
For this PR, there is no significant difference in performance. Could you please take a look at the latest 2 commits in this https://github.com/apache/beam/pull/4741 ? 
My idea here is:
Every time trying to check experimental options by calling [RuntimeValueProvider.get_value()](https://github.com/apache/beam/blob/master/sdks/python/apache_beam/options/value_provider.py#L82), it will cost O(n), where n is the # of options. For https://github.com/apache/beam/pull/4741 it's kind of necessary since we call it per element. For this PR, I wanna keep the same pattern. 
Maybe it's the best idea to do this, could you explain more about why 'RuntimeValueProvider is not the right place to have attributes for different options.' ?